src/main/scala/com/typesafe/sbteclipse/core/Eclipse.scala

```diff
@@ -393,8 +393,8 @@ private object Eclipse extends EclipseSDTConfig {
       case None => JreContainer
     }
 
-  def builderAndNatures(projectFlavor: EclipseProjectFlavor.Value) =
-    if (projectFlavor.id == EclipseProjectFlavor.ScalaIDE.id)
+  def builderAndNatures(projectFlavor: EclipseProjectFlavor.Value): (String, Seq[String]) =
+    if (projectFlavor == EclipseProjectFlavor.ScalaIDE)
```

Comment:
did you test this? I'm not sure why, but we've found it very necessary to have the IDs here. it took @jsuereth and myself quite awhile to track this down. i don't think you can just remove them